### ISSUE TITLE
ERROR at //build/config/posix/BUILD.gn:56:28: Undefined identifier in string expansion.

### ISSUE THREAD

**Issue by anchev** [Eloston/ungoogled-chromium#566]
### Description

I am getting this error during build and further building is not possible.

### Console output and logs

```
ninja: Entering directory `/tmp/download/ungoogled-chromium/build/src/out/Release/gn_build'
[171/171] LINK gn
+ ./out/Default/gn gen out/Default --fail-on-unused-args
ERROR at //build/config/posix/BUILD.gn:56:28: Undefined identifier in string expansion.
      "CR_LIBCXX_REVISION=$libcxx_svn_revision",
                           ^------------------
"libcxx_svn_revision" is not currently in scope.
See //build/config/compiler/BUILD.gn:1192:18: which caused the file to be included.
    configs += [ "//build/config/posix:runtime_library" ]
                 ^-------------------------------------
```

### Environment details

* ungoogled-chromium version: 70.0.3538.67
* Configuration bundle: `linux_portable`
* OS and version for building: openSUSE Leap 15.0
* Compiler toolchain:
```
$ clang -v
clang version 8.0.0 (trunk 345148)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /<***>/llvm-build/bin
Found candidate GCC installation: /usr/lib64/gcc/x86_64-suse-linux/7
Selected GCC installation: /usr/lib64/gcc/x86_64-suse-linux/7
Candidate multilib: .;@m64
Selected multilib: .;@m64
```

### Building and packaging steps

The same described in https://github.com/Eloston/ungoogled-chromium/issues/447 which so far have worked fine (currently running 69.0.3497.100 built the same way).

**Comment by Eloston** [comment]
This issue gave me deja-vu, until I discovered that I actually solved this problem before.

@xsmile Just a heads up: I have made a few changes to Debian patches when I include them. In this issue, [I needed to remove a section unrelated to fontconfig from `debian/system/fontconfig.patch`](https://github.com/Eloston/ungoogled-chromium/blob/93e5400cfdae265e59e91046fa80515c69f7139b/patches/debian/system/fontconfig.patch#L4) that was causing this error. I will make a note in the patch itself so we won't make the same mistake next time.

I will fix this. Thanks for reporting @anchev.
